analytics/src/analytics_desktop.cc

```diff
@@ -134,6 +139,10 @@ bool IsInitialized() { return g_initialized; }
 void Terminate() {
 #if defined(_WIN32)
   if (g_analytics_module) {
+    // Make sure to notify the SDK that the analytics is being terminated to
+    // upload any pending data.
+    NotifyAppLifecycleChange(AppLifecycleState::kTermination);
```

Comment:
I am unclear if this is something we want to automate, or leave to the app developer. Would be good to confirm.